packages/meta-harness/tests/test_scenario_isolation.py

```
@@ -232,6 +233,121 @@ def test_container_git_wrapper_uses_fixed_image_git_not_itself(
+    collateral-scope oracle's unconditional symlink rejection can catch a candidate that plants
+    `ln -s <dir> .claude/docs/evil-link` inside an already-tracked-but-ignored directory) without
+    following it into the linked-to directory's contents."""
+    gitignore = git_project / ".gitignore"
+    gitignore.write_text(
+        gitignore.read_text(encoding="utf-8") + ".claude/docs/\n",
+        encoding="utf-8",
+    )
+    plan = git_project / ".claude" / "docs" / "plan.md"
+    plan.parent.mkdir(parents=True)
+    plan.write_text("tracked despite gitignore\n", encoding="utf-8")
+    subprocess.run(["git", "add", ".gitignore"], cwd=git_project, check=True)
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)